public/app/features/plugins/admin/components/VersionList.tsx

```diff
@@ -1,36 +1,59 @@
 import { css } from '@emotion/css';
```

Comment:
This component is now much more complex than a version list, please add unit tests for it.